docs/DafnyRef/Types.md

```diff
@@ -4495,4 +4495,65 @@ imposed on the body of `ReachableVia` makes sure that, if the
 predicate returns `true`, then every object reference in `p` is as old
 as some object reference in another parameter to the predicate.
 
+## 6.5. Nameonly Formal Parameters and Default-Value Expressions
+
+A formal parametes of a method, constructor in a class, iterator,
+function, or datatype constructor can be declared with an expression
+denoting a _default value_. This makes the parameter _optional_,
+as opposed to _required_. All required parameters must be declared
+before any optional parameters. 
```

Comment:
No, this is no longer true (once https://github.com/dafny-lang/dafny/pull/3864 is merged). A required parameter is allowed to follow an optional parameter, provided the required parameter is `nameonly`. (It may be best to omit the sentence here and explain this below, after `nameonly` has been introduced.)

Actually, L4538 says what needs to be said.

Reply:
Removed the sentence.